crates/bevy_core_pipeline/src/prepass/node.rs

```diff
@@ -162,7 +162,7 @@ impl ViewNode for PrepassNode {
             pass_span.end(&mut render_pass);
             drop(render_pass);
 
-            // Copy prepass depth to the main depth texture if deferred isn't going to
+            // After rendering to the view depth texture, copy it to the prepass depth texture if deferred isn't going to
```

Comment:
The comment makes sense for what the code does, but I'm not sure I understand when that would ever happen. If you use deferred + prepass, then you're using deferred so it will do the copy. If you just use the prepass without deferred, it should run before the main pass so I don't get why we would want to copy the view depth to the prepass depth. Shouldn't it be the other way around?

Reply:
Oh, is it because the meshlet prepass writes directly to the view depth?

Reply:
I think meshlets should be mentioned in that comment if that's the case.

Reply:
This is existing code, for the regular (non meshlet) prepass. We don't actually render directly to the prepass texture, we render to the view depth, and then copy the result into the prepass depth texture.

Reply:
Oh, right, I remember now.